packages/flutter/test/widget/bottom_sheet_rebuild_test.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/material.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('Verify that a BottomSheet can be rebuilt with ScaffoldFeatureController.setState()', () {
+    testWidgets((WidgetTester tester) {
+      final GlobalKey<ScaffoldState> scaffoldKey = new GlobalKey<ScaffoldState>();
+      ScaffoldFeatureController bottomSheet;
+      int nBottomSheetBuilds = 0;
```

Comment:
nit: Please use complete words in variable names (e.g., numberOfBottomSheeBuilds)